.rnucrc.js

```diff
@@ -0,0 +1,28 @@
+var ncp = require('ncp').ncp;
+
+module.exports = {
+  on_env: async function (env) {
+    // Copy Android res folder
+    const resSourceFolder = `./android/app/src/${env.RES_ID}/`;
+    const resDestFolder = "./android/app/src/main/";
+    ncp(
+      resSourceFolder,
+      resDestFolder,
+      err => {
+        if (err) {
+          return console.error(err);
+        }
+        console.log('copied android resources!');
+      })
+
+    // Copy iOS icon set
+    const xcassetsSourceFolder = `./ios/RedVSBlue2/Images.xcassets/${env.ICON_SET_NAME}.appiconset/`;
+    const xcassetsDestFolder = './ios/RedVSBlue2/Images.xcassets/AppIcon.appiconset/';
+    ncp(xcassetsSourceFolder, xcassetsDestFolder, (err) => {
```

Comment:
this is a bit more tricker than android (depending on goal of course)

I believe the real gain here is to completely extract icons from shipped application. This approach will have *both* icons in the final app. In fact as long as having both icons within "ipa" file is not a concern one can already achieve this with something as s
<img width="631" alt="Screen Shot 2020-07-20 at 10 36 15 PM" src="https://user-images.githubusercontent.com/7002833/87978655-7782db80-cad9-11ea-831d-49ce721777a5.png">
imple as:
<img width="614" alt="Screen Shot 2020-07-20 at 10 36 11 PM" src="https://user-images.githubusercontent.com/7002833/87978653-76ea4500-cad9-11ea-9ece-767ad2289fa5.png">

To make sure no unnecessary assets are shipped with the app it would require to maintain 2 asset catalogs. one as "source" - it should be removed from all targets. and a "target" catalog which will be copied to binary at build. so the rnuc hook could copy files *between* catalogs, not *within* same catalog.

Reply:
Good point! I (wrongly) assumed not all images would be packed into the ipa. 

> In fact as long as having both icons within "ipa" file is not a concern one can already achieve this with something as [...]

Yes, you are right! Your example was the setup before making the changes in this PR.

> To make sure no unnecessary assets are shipped with the app it would require to maintain 2 asset catalogs. one as "source" - it should be removed from all targets. and a "target" catalog which will be copied to binary at build. so the rnuc hook could copy files between catalogs, not within same catalog.

I've added an `assets` folder and updated the rnuc hook script to copy the intended assets from there for both android and ios. I believe this should effectively remove the redundant resources from the builds, with the added benefit of having all white label assets in a more manageable location.

What do you think?

Reply:
yeah, that's one way to do it.
I was thinking about having 2 asset catalogs though: the reason is that for ios you most likely want to edit these things via xcode. but your solution definitely looks cleaner.

Reply:
I see! In my case you would have to go through a little more manual effort to update and replace the separate folders. I'm not very experienced in XCode, working more around it than with it in my RN projects :-)